libvncclient/rfbproto.c

```diff
@@ -1586,6 +1589,30 @@ SendKeyEvent(rfbClient* client, uint32_t key, rfbBool down)
 }
 
 
+/*
+ * SendExtendedKeyEvent.
+ */
+
+rfbBool
+SendExtendedKeyEvent(rfbClient* client, uint32_t keysym, uint32_t keycode, rfbBool down)
+{
+  rfbQemuExtendedKeyEventMsg ke;
+
+  // FIXME: rfbQemuEvent also covers audio events, but this model for checking
```

Comment:
Sry, one more minor thing: can you convert those to C-style comments please?

Reply:
Done